go.mod

```diff
@@ -53,12 +53,12 @@ require (
 	github.com/gofiber/utils v0.0.10
 	github.com/gofrs/uuid v3.3.0+incompatible
 	github.com/golang-migrate/migrate v3.5.4+incompatible
-	github.com/golang/mock v1.4.4
+	github.com/golang/mock v1.5.0
```

Comment:
what is the need for upgrading these package versions? Please do not touch .mod and .sum files

Reply:
Fixed